commitlint.config.js

```diff
@@ -1,6 +1,20 @@
 module.exports = {
-    rules: {
-      'signed-off-by': [2, 'always', 'Signed-off-by:'],
-    },
-  }
+  rules: {
+    'signed-off-by': [2, 'always', 'Signed-off-by:'],
+    'type-enum': [2, 'always', [
+      'feat',
+      'fix',
+      'docs',
+      'style',
+      'refactor',
+      'perf',
+      'test',
+      'chore',
+      'revert',
+    ]],
```

Comment:
conventional commit check essentially - has to include one of these types ie. `chore: blah`